server/migrations/20171226132051-create-user.js

```diff
@@ -0,0 +1,27 @@
+
+module.exports = {
```

Comment:
In any file that that the `.js` extension, instead of `module.exports = { }`, use `export default { }`